nautobot/utilities/testing/views.py

```diff
@@ -291,31 +291,37 @@ def test_get_object_with_permission(self):
             response = self.client.get(instance.get_absolute_url())
             self.assertHttpStatus(response, 200)
 
+            response_body = extract_page_body(response.content.decode(response.charset))
+
             # The object's display name or string representation should appear in the response
-            self.assertIn(getattr(instance, "display", str(instance)), str(response.content))
+            self.assertIn(getattr(instance, "display", str(instance)), response_body, response_body)
```

Comment:
Since it's not immediately apparent (and I had to look it up), would you mind making the use of the optional 3rd arg be obvious that it's the `msg=` param?
```suggestion
            self.assertIn(getattr(instance, "display", str(instance)), response_body, msg=response_body)
```

Reply:
Great suggestion. I hadn't remembered that it could be a kwarg.